ui/components/app/custom-spending-cap/index.scss

```diff
@@ -21,11 +21,22 @@
       position: absolute;
       margin-top: 55px;
       margin-inline-start: -75px;
+      z-index: 1;
     }
   }
 
   #custom-spending-cap-input-value {
     color: var(--color-error-default);
     padding-inline-end: 60px;
   }
+
+  input[type='number']::-webkit-inner-spin-button {
+    -webkit-appearance: none;
+    display: none;
+  }
+
+  input[type='number']:hover::-webkit-inner-spin-button {
+    -webkit-appearance: none;
+    display: none;
+  }
```

Comment:
This can be:

```
input[type='number']::-webkit-inner-spin-button,
input[type='number']:hover::-webkit-inner-spin-button {
    -webkit-appearance: none;
    display: none;
  }
```

Reply:
Fixed here: 7be0fcc2ad8e2939392b9cb50a3f95e0432cd76e